### PR TITLE
Make app more dev friendly

### PR DESCRIPTION
**Pull request by benilovj**
### Context

This app doesn't have any documentation on how to set it up for development, nor any test data to get developers going.

### Changes proposed in this pull request

* Add a README
* Add some seed test data

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Support teacher training publishers
+
+This app is used by the user support team to help publishers with the Pubish teacher training courses service.
+
+## Setting up the app in development
+
+1. Create a Postgresql user:
```
Comment (dankmitchell) [inline README.md]: I really don’t like the idea of of this in development. Can we just go with a vanilla setup and only set username in production?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    ```
+
+2. Run
+
+    ```
+    bundle exec rake db:setup
+    ```
+    to set up the database development and test schemas, and seed with test data.
+
+3. Run `./start.sh` to launch the app on http://localhost:3000
```
Comment (dankmitchell) [inline README.md]: Can we just run `rails s`?

File: README.md (hunk 1 above)
Comment (benilovj) [reply README.md]: Yeah no worries – I've updated the PR with this.

File: README.md (hunk 2 above)
Comment (benilovj) [reply README.md]: Fair enough – simplified this.

**Pull request merged**